.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "src/lib/snarky"]
 	path = src/lib/snarky
 	url = https://github.com/o1-labs/snarky
-[submodule "src/lib/crypto/proof-systems"]
```

Comment:
Nice :) personal take but submodule structure is annoying. For (the rare) debugging the two apps together you can always override the dependency to a local copy anyways.

Reply:
We could also do better versioning for `proof-systems`... at some point.

Reply:
You're gonna hate, but I had to reintroduce it :(, see [here](de71aa218db6d87fba49f827ccb0123907f4b7c5).